ArkEcosystem.Crypto.Tests/Transactions/TransactionTest.cs

```diff
@@ -25,7 +25,7 @@
 using NBitcoin.DataEncoders;
 using System;
 
-namespace ArkEcosystem.Crypto.Tests.Configuration
+namespace ArkEcosystem.Crypto.Tests
```

Comment:
Should be `namespace ArkEcosystem.Crypto.Tests.Transactions` since your test is in the `Transactions` sub-directory.

Reply:
Sorry about that Brian.  Eyes played tricks on me and misread the file tree, lol.  Fix incoming.